ext/span.c

```diff
@@ -163,7 +163,7 @@ void ddtrace_open_span(ddtrace_span_data *span) {
 set_trace_id_from_span_id:
         span->trace_id = (ddtrace_trace_id){
             .low = span->span_id,
-            .high = get_DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED() ? span->start : 0,
+            .time = get_DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED() ? span->start / 1000000000L : 0,
```

Comment:
I think you want `UINT64_C` instead of `L`:

```suggestion
            .time = get_DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED() ? span->start / UINT64_C(1000000000) : 0,
```

I'm not feeling well today, but this is going from nanoseconds down to seconds, I think? Just checking that's the intention? I'd add a comment about the unit in `ddtrace_trace_id.time`'s definition.

Reply:
I did just copy the L from somewhere else and it had the expected value :-D But UINT64_C is more correct, let me do that.